KlingonAssistant/data/mem-05-H.xml

```diff
@@ -2065,7 +2065,7 @@ Watch Gowron say this: {YouTube video:url:http://www.youtube.com/v/KArQlviVXMk}<
       <column name="entry_name">Hoch nuH qel</column>
       <column name="part_of_speech">sen:idiom</column>
       <column name="definition">to consider every possibility</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">jede Möglichkeit in Erwägung ziehen</column>
```

Comment:
What editor are you using? It's inserting newlines, probably at 80 characters. Newlines in the entries will mess up the database.